libraries/botframework-schema/src/index.ts

```diff
@@ -1226,6 +1226,29 @@ export interface TokenResponse {
   expiration: string;
 }
 
+/**
+ * The status of a particular token
+ */
+export interface TokenStatus {
```

Comment:
This is an auto-generated file, if I remember correctly. This class will be deleted the next time we generate from the swagger files. 

Could you confirm this will work properly as part of the swagger regeration?

Reply:
Note - this should be the same as C#, regardless of the status. Whatever we've done there, is the same solution we should use here.

Reply:
This has been removed as it should not have been changed